sysinfo.ts

```diff
@@ -64,6 +66,26 @@ export class SysInfo implements ISysInfo {
 		} // if we got an error, assume not working
 	}
 
+	// `android -h` returns exit code 1 on successful invocation. Therefore, we cannot use $childProcess
```

Comment:
Is this unexpected behavior detected on MacOS only? I've tried to reproduce it on Windows, but without success.